mmv1/third_party/terraform/services/securitycenter/resource_scc_organization_big_query_export_config_test.go

```diff
@@ -34,7 +47,8 @@ func TestAccSecurityCenterOrganizationBigQueryExportConfig_basic(t *testing.T) {
 		},
 		Steps: []resource.TestStep{
 			{
-				Config: testAccSecurityCenterOrganizationBigQueryExportConfig_basic(context),
+				Config:  testAccSecurityCenterOrganizationBigQueryExportConfig_basic(context),
+				Destroy: true,
```

Comment:
What is this line meant to be doing? I suspect that's why we're failing the tests- we're removing all the resources from state and therefore fail to find them.

Reply:
Addressed. Thank you for the suggestion, this is causing the issue.